src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -268,22 +268,10 @@ def get_suggested_assignees(
             if assignee.actor_type == ActorType.USER and not (
                 isinstance(current_assignee, RpcUser) and assignee.id == current_assignee.id
             ):
-                # for user assignees, we first try to get their Slack identity; if it's not linked,
-                # we use their display name linked with their email
-                assignee_identity = None
-                assignee_identities = identity_service.get_user_identities_by_provider_type(
-                    user_id=assignee.id, provider_type=ExternalProviderEnum.SLACK.value
+                assignee_as_user = assignee.resolve()
+                assignee_texts.append(
+                    f"<mailto:{assignee_as_user.email}|{assignee_as_user.get_display_name()}>"
```

Comment:
I think we just want it as regular text and not a link but I'll check with Dhrumil

Reply:
Ok yeah confirmed - for the suspect commits I prioritized the display name and fell back to email